pages/workshops.tsx

```diff
@@ -11,53 +11,39 @@ import { useQuery } from "@apollo/client";
 import GoalsSectionComponent from "../components/coding/GoalsSectionComponent";
 import { Button } from "../components/ui/Button";
 import { useRouter } from "next/router";
+import WorkshopsCard from "../components/coding/studentPortal/WorkshopsCard";
 
-export default function Goals(props) {
+export default function Workshops(props) {
   const { user } = useAuth();
   const router = useRouter();
 
   return (
     <div className="flex flex-col p-4 m-4 overflow-auto bg-scroll">
       <div>
         <div className="mb-8 text-3xl">Workshops</div>
+
         <div className="grid grid-cols-1 gap-8 sm:grid-cols-2">
-          <div>
-            <h2>Course HTML/CSS Page</h2>
-            <video
-              src="https://d3jppm0n5ndqu2.cloudfront.net/header-section-workshop.mp4"
-              controls={true}
-            />
-            <a href="https://d3jppm0n5ndqu2.cloudfront.net/course-workshop-files.zip">
-              <div className="mt-4">
-                <Button label={"Starter Files"} />
-              </div>
-            </a>
-          </div>
-          <div>
-            <h2>War Game: JS Basics - August 8, 2022</h2>
-            <video
-              src="https://d3jppm0n5ndqu2.cloudfront.net/war-workshop.mp4"
-              controls={true}
-            />
-          </div>
-          <div>
-            <h2>Pokedex - July 20, 2022</h2>
-            <video
-              src="https://d3jppm0n5ndqu2.cloudfront.net/pokedex-workshop-final.mp4"
-              controls={true}
-            />
-          </div>
-          <div>
-            <h2>Pokedex - July 21, 2022</h2>
-            <video
-              src="https://d3jppm0n5ndqu2.cloudfront.net/pokedex-july-21.mov"
-              controls={true}
-            />
-          </div>
+          <WorkshopsCard
```

Comment:
You can clean up this code by creating an array of WorkshopsCardProps and putting all the values in that array.

Then you can map through that array to render a list of the card components.

What you have is correct but my suggestion is a better practice for cleaner code.

Reply:
Look at this file for example